torch/distributed/distributed_c10d.py

```diff
@@ -493,7 +509,14 @@ def _new_process_group_helper(world_size,
             _pg_map[pg] = (Backend.NCCL, store)
             _pg_names[pg] = group_name
         else:
-            raise RuntimeError("Unsupported distributed backend by group")
+            pg_name = "ProcessGroup{}".format(backend.upper())
+            pg = globals()[pg_name](
+                prefix_store,
+                rank,
+                world_size,
+                group_name)
+            _pg_map[pg] = (pg_name, store)
```

Comment:
yes, it's an assumption that module name 'xyz' can be used to infer usable interface ‘ProcessGroupXYZ’. it's because we have to have a rule to predict such interface symbol...

Reply:
@ftian1 I think what @mrshenli meant was that whether it is fine to use `("xyz", store)` instead of `("ProcessGroupXYZ", store)` in `_pg_map[pg]` since other backends use the former naming convention. You can still use `globals()["ProcessGroupXYZ"]` to get `pg` here.